packages/UI/src/components/atoms/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,60 @@
+import * as Avatar from '@radix-ui/react-avatar';
+
+import { tv, type VariantProps } from 'tailwind-variants';
+
+const AvatarStyles = tv({
+  base: ' h-12 w-12 justify-center   rounded-full bg-white  bg-contain text-center align-middle text-base font-medium',
+});
+
+interface AvatarProps
+  extends VariantProps<typeof AvatarStyles>,
+    Avatar.AvatarProps {
+  src: string;
+  fallback: React.JSX.Element;
+  alt: string;
+  delayMs?: Avatar.AvatarFallbackProps['delayMs'];
+  onLoadingStatusChange?: Avatar.AvatarImageProps['onLoadingStatusChange'];
```

Comment:
it would be nice to have the props with description based on what https://www.radix-ui.com/primitives/docs/components/avatar already describes them